### PR TITLE
add Team page

### PR DESCRIPTION
**Pull request by andyyPark**
Sorry for the delay but this PR shows the template of our team page. The code I pushed has 2 rows but I used only one row to take the screenshot. Let me know what you guys think! There is also a gif at the bottom of the comment section

image

### CODE REVIEW

**Comment by TDHTTTT** [comment]
So you want to have 2 rows but only got 1? Lazy thought: Also, can we have a "dev" branch which can also be served online as a live review? (so that we don't have to do this locally)

**Comment by andyyPark** [comment]
Sorry I meant to say that the code I pushed has 2 rows but I used one row to take the screenshot. Also what do you mean by live review?

**Comment by TDHTTTT** [comment]
I see. NVM, hosting locally for reviewing dev might be a better approach.

File: css/style.css
Comment (TDHTTTT) [inline css/style.css]: Maybe a darker white?

File: css/style.css
Comment (TDHTTTT) [inline css/style.css:267]: Same, maybe a darker white?

**Comment by TDHTTTT** [comment]
So are we going for a light theme or maybe a dark theme?

**Comment by TDHTTTT** [comment]
Also, maybe it would be good to have sections in the Team page as @omarcostahamido suggested in Notion? @andyyPark 

- current board members
- previous board members
- founding members

File: css/style.css
Comment (andyyPark) [reply css/style.css]: Do you have a specific dark white you want me to use?

**Comment by Chriscrosser3310** [comment]
I would suggest having a darker theme since it's more consistent with the home page.

**Review by Chriscrosser3310 — APPROVED** [review]

**Comment by andyyPark** [comment]
I added a filter for current/previous/founding members @omarcostahamido @TDHTTTT. Here is a gif!

ezgif com-video-to-gif

**Comment by TDHTTTT** [comment]
Good job! @andyyPark I noticed that the scroll bar jumps from "Current Board" to other view since it has overflow, I found a quick fix [here](https://aykevl.nl/2014/09/fix-jumping-scrollbar) to force the scroll bar everywhere.

**Comment by andyyPark** [comment]
Thanks for the fix @TDHTTTT ! What browser were you using?

**Comment by TDHTTTT** [comment]
@andyyPark Firefox 80.0.1

**Comment by omarcostahamido** [comment]
Screen Shot 2020-09-09 at 2 17 03 AM
(using chrome)

if possible, please review the break points.

Also, I'd say move the current/previous/founding tabs to center.

And though I think for now it will just work, in the future, the previous will have to account for multiple years. So, an alternative is to call current "2020-2021" and previous "2019-2020". But don't stress too much about this rn. We can just add as a todo task for next year.

I'm ok if others want to have their email up but I'd prefer not having mine up. Also, I don't have linkedin.

Questions: 
  - is there a github logo we can add? Also some other ppl could prefer facebook as well.
  - is it interesting to add a slack icon as well? (not sure this question makes sense, but in my mind it would something that would link to our slack username... so people reach out and interact more on slack that is)
  - what do you have in mind for that lorem ipsum placeholder text?

We need to keep thinking about consistency in design so I would suggest making the executive decision that all board member photos submitted would have to be square. Sometimes the round shapes photos are also nice, do you want to try it?

**Review by omarcostahamido — CHANGES_REQUESTED** [review]
Screen Shot 2020-09-09 at 2 17 03 AM
(using chrome)

if possible, please review the break points.

Also, I'd say move the current/previous/founding tabs to center.

And though I think for now it will just work, in the future, the previous will have to account for multiple years. So, an alternative is to call current "2020-2021" and previous "2019-2020". But don't stress too much about this rn. We can just add as a todo task for next year.

I'm ok if others want to have their email up but I'd prefer not having mine up. Also, I don't have linkedin.

Questions:

  - is there a github logo we can add? Also some other ppl could prefer facebook as well.
  - is it interesting to add a slack icon as well? (not sure this question makes sense, but in my mind it would something that would link to our slack username... so people reach out and interact more on slack that is)
  - what do you have in mind for that lorem ipsum placeholder text?

We need to keep thinking about consistency in design so I would suggest making the executive decision that all board member photos submitted would have to be square. Sometimes the round shapes photos are also nice, do you want to try it?

**Comment by omarcostahamido** [comment]
In what regards to colors, the only thing I would like to say is that I like that color scheme from the emails.
You can literally look at the code here: https://www.qc-uci.club/site_old/ & https://github.com/QC-UCI/site_old
that background is this very dark grey: #242637

**Comment by andyyPark** [comment]
Fixed the break points and changed the background color to match the join page @omarcostahamido

**Review by omarcostahamido — APPROVED** [review]

**Comment by omarcostahamido** [comment]
Thanks. I removed @Ahmad-Fermion @DanielQ56 and @KylerFrazier from the list of required approval, to see if we can merge this soon.

@TDHTTTT and @Chriscrosser3310 can you review and approve the latest changes?
thank you

**Comment by andyyPark** [comment]
I was thinking of going over the two PRs and merge them during the meeting. Should we merge beforehand?

**Comment by omarcostahamido** [comment]
@andyyPark your call. But the necessary discussion we've been pretty much having it here already.
I'd say if anything still needs to change ---> [create issue](https://github.com/QC-UCI/QC-UCI.github.io/issues/new). And we'll deal with that.
(I just want to move forward.)

**Review by TDHTTTT — COMMENTED** [review]
As for the officer picture and short bio, let's ask everyone in the Notion for those assets.

**Comment by TDHTTTT** [comment]
@andyyPark i don't think you need to merge join -> team. In the end everything just goes to master.

**Comment by omarcostahamido** [comment]
@andyyPark just a reminder that there is content ready to move here on the [notion page](https://www.notion.so/Picture-short-bio-and-contact-for-the-website-2aafcb2a500b4a71970ed04f1d8d351c).

in the mean time I'll start drafting some text [here](https://www.notion.so/Set-up-a-website-f7858bd01d0c4b89855f1274f303a156) to add after the list of members, for each tab. Possible?

**Comment by omarcostahamido** [comment]
Finished collecting all the available content for this page [here](https://www.notion.so/Set-up-a-website-f7858bd01d0c4b89855f1274f303a156).
@andyyPark You just have to copy-paste and figure out how to deal with the different image aspect ratios and the repeated information - which I suggested [here](https://www.notion.so/Set-up-a-website-f7858bd01d0c4b89855f1274f303a156#fd6e3795a38d4a27a97683447775fd8c) to be set as a pop up.

**Comment by andyyPark** [comment]
Thank you for the organization @omarcostahamido! I will have it up soon

**Comment by andyyPark** [comment]
I updated all the pictures of the members but as you can see I still have to adjust the aspect ratio. Any tips on this?

**Comment by omarcostahamido** [comment]
either crop them before uploading it, or use something else in css like [this](https://www.educative.io/edpresso/how-to-crop-an-image-in-css).

**Comment by andyyPark** [comment]
This fixes #10

**Comment by omarcostahamido** [comment]
@andyyPark good job.

**Comment by omarcostahamido** [comment]
@andyyPark I noticed that you didn't include Harry. Can you still add him, with no picture or a place holder picture?

**Comment by omarcostahamido** [comment]
Also, the collected bios are all missing too. I will create a new issue for that.

**Comment by andyyPark** [comment]
@omarcostahamido Just added him!

**Comment by omarcostahamido** [comment]
thanks @andyyPark

**Pull request merged**
